weblab/entities/migrations/0016_delete_old_style_fitting_specs.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2020-11-25 15:02
+
+from django.db import migrations
+
+
+def delete_protocols(apps):
+    Protocol = apps.get_model('entities', 'ProtocolEntity')
+    for protocol in Protocol.objects.filter(is_fitting_spec=True):
+        protocol.delete();
```

Comment:
We don't need ; in Python :)

Reply:
apologies... I tried to retest but not sure how to "rollback" migrations. when I try and specify an earlier migration I get error message 

`django.db.migrations.exceptions.InconsistentMigrationHistory: Migration entities.0017_remove_entity_is_fitting_spec is applied before its dependency entities.0016_delete_old_
style_fitting_specs on database 'default'.`

Reply:
Because we didn't provide a reverse migration for 0016 it isn't reversible, so you can't go back to before that state. I'll test on a VM here, but looks like it should be OK now.